gcloud-java-bigquery/src/main/java/com/google/gcloud/bigquery/ExtractJobConfiguration.java

```diff
@@ -200,7 +199,7 @@ public Builder toBuilder() {
   }
 
   @Override
-  ToStringHelper toStringHelper() {
+  protected ToStringHelper toStringHelper() {
```

Comment:
+1

Though I think it is more than reasonable to declare methods as protected when we expect sub-classes to override them (and the class is public) I consider toStringHelper, baseHashCode, baseEquals to be more of an internal (package-scope) details, not to mention that I would prefer if they don't show up in our javadoc.